projects/sunbird-video-player/src/lib/components/video-player/video-player.component.spec.ts

```diff
@@ -1,10 +1,12 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { HttpClientModule } from '@angular/common/http';
 import { VideoPlayerComponent } from './video-player.component';
+import videojs from 'video.js';
```

Comment:
instead of importing here similar to app add this dependency in angular.json **test** config for the library project